docs/docs/user-guide/training-data-format.mdx

```diff
@@ -0,0 +1,659 @@
+---
+id: training-data-format
+sidebar_label: Training Data Format
+title: Training Data Format
+description: Description of the YAML format for training data
+---
+
+## Overview
+
+Rasa Open Source uses [YAML](https://yaml.org/spec/1.2/spec.html) as
+a unified and extendable way to manage all training data,
+including NLU data, stories and rules. 
+
+With the YAML format, training data can be split over any number of YAML files,
+and every file can contain any kind of data. 
+The training data parser will read the top level keys in each file to decide
+what kind of data is in a section at training time.
+
+The [domain](../glossary#domain) uses
+the same YAML format as the training data and can also be split across
+multiple files or combined with training data in one file. The domain includes
+the definitions for [responses](../core/domains#responses) and [forms](../core/forms).
+See the [documentation for domains](../core/domains) to learn more.
```

Comment:
@m-vdb i'm confused, do these links need to have `.mdx` in them or not? Running them locally with the `make install-docs` from `master` and the links including the anchors seem to render fine 🤔

Reply:
I think it should be according to the docusaurus docs: https://v2.docusaurus.io/docs/markdown-features/#referencing-other-documents

Reply:
you don't have too, but it looks like it's preferred (the only benefit I see from now is that if we want to navigate the doc from github files, but in our case it wouldn't really work anyway).